packages/bsky/src/api/blob-resolver.ts

```diff
@@ -85,16 +85,22 @@ export async function resolveBlob(
 ) {
   const cidStr = cid.toString()
 
-  const [{ pds }, takedown] = await Promise.all([
+  const [{ pds }, takedownStatus, takedown] = await Promise.all([
     idResolver.did.resolveAtprotoData(did), // @TODO cache did info
     db.db
       .selectFrom('moderation_subject_status')
       .select('id')
       .where('blobCids', '@>', sql`CAST(${JSON.stringify([cidStr])} AS JSONB)`)
       .where('takendown', 'is', true)
       .executeTakeFirst(),
+    db.db
+      .selectFrom('blob_takedown')
+      .select('takedownId')
+      .where('did', '=', did)
+      .where('cid', '=', cid.toString())
+      .executeTakeFirst(),
```

Comment:
I imagine we'll eventually get rid of one of these checks.

Reply:
uhh yes good call - actually i should just nix it now 👍 

I think we'll want to keep both checks in the intermediate branch